net-fs/nfs-utils/nfs-utils-2.6.3-r4.ebuild

```diff
@@ -19,8 +19,8 @@ fi
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="caps junction kerberos ldap +libmount nfsdcld +nfsidmap +nfsv4 nfsv41 sasl selinux tcpd +uuid"
-REQUIRED_USE="kerberos? ( nfsv4 )"
```

Comment:
dont change the stable ebuild, let's leave stable for the time being, since this is a bit invasive

Reply:
Ok.